plugins/onepad/joystick.cpp

```diff
@@ -218,7 +218,7 @@ bool JoystickInfo::Init(int id)
 	// Keep the 4 hat buttons too (usb driver). (left pressure does not work with recent kernel). Moreover the pressure
 	// work sometime on half axis neg others time in fulll axis. So better keep them as button for the moment
 	u32 found_hack = devname.find(string("PLAYSTATION(R)3"));
-	if (found_hack != string::npos) {
+	if (found_hack != string::npos && (conf->options & PADOPTION_SIXAXIS_USB) == 0) {
```

Comment:
Yes, if the DS3 USB box is unchecked the the current behavior of ignoring the toggle events should be maintained.  If however  the box is checked the the axis are unavailable and the toggle events should be processed.